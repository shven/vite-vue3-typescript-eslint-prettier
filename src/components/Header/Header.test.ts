import { mount } from '@vue/test-utils';
import Component from './VueHeader.vue';

describe('VueHeader.vue', () => {
    const wrapper = mount(Component);

    it('Match snapshot', () => {
        expect(wrapper.element).toMatchSnapshot();
    });
});
