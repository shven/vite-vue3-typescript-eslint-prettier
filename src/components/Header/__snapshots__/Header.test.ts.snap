// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`VueHeader.vue > Match snapshot 1`] = `
<div
  class="header"
  data-v-a1588586=""
>
  <div
    class="container"
    data-v-a1588586=""
    data-v-cbb0d1af=""
  >
    
    <div
      class="header__content"
      data-v-a1588586=""
    >
      <nav
        class="header__nav"
        data-v-a1588586=""
      >
        
        <routerlink
          data-v-a1588586=""
          to="/"
        >
          Home
        </routerlink>
        <routerlink
          data-v-a1588586=""
          to="/styleguide"
        >
          Styleguide
        </routerlink>
        <routerlink
          data-v-a1588586=""
          to="/movies"
        >
          Movies
        </routerlink>
        <routerlink
          data-v-a1588586=""
          to="/photos"
        >
          Photos
        </routerlink>
        
      </nav>
    </div>
    
  </div>
</div>
`;
